### ISSUE TITLE
Annotations.findJsonAnnotation will crash on Android

### ISSUE THREAD

**Issue by vishna** [cbeust/klaxon#93]
https://github.com/cbeust/klaxon/blob/7e2ae8b0baf7eb7ace1dbca64c799f053a7b1fc2/src/main/kotlin/com/beust/klaxon/Annotations.kt#L15

Method getDeclaredAnnotation() doesn't exist on Android

https://developer.android.com/reference/java/lang/reflect/Field.html

**Comment by cbeust** [comment]
Good catch, going to replace it with:

```
val r1 = kc.java.getDeclaredField(propertyName).annotations.firstOrNull { it.javaClass == Json::class.java }
```

**Comment by cbeust** [comment]
Fixed in 2.1.1.
